test/webpack.test.ts

```diff
@@ -18,7 +18,7 @@ describe('webpack', () => {
       mkdirSync(resolve(templateDir, 'build'));
       return runCommands([
         `cd ${templateDir}`,
-        'yarn build'
+        'yarn build --bundleAnalyzer'
```

Comment:
what does this change for the test? Is it testing the `bundleAnalyzer` setup in some way? It doesn't seem so, I would remove the flag from here

Reply:
Leftover